packages/module/patternfly-docs/content/design-guidelines/design-guidelines.md

```diff
@@ -9,6 +9,22 @@ id: User feedback
 source: design-guidelines
 ---
 
-Design guidelines
+Note: User feedback lives in its own package at [@patternfly/react-user-feedback](https://github.com/patternfly/react-user-feedback).
 
-COMING SOON!
+**User feedback** is a modal component that on-premises products can use to collect asynchronous feedback from users. 
+
+## Behavior
+
+When installed, user feedback may be accessed in a couple of ways: 
```

Comment:
```suggestion
The user feedback modal may be accessed in a couple of ways: 
```

Are PF extensions "installed?" Also clarified that it's a modal here.

Reply:
Yeah I wasn't sure about saying "installed", but I also am not myself familiar with the process of using extensions so I took a guess there. Either way it's probably plenty obvious that you have to add the modal to your product to access it anyways, so your suggestion sounds great!